### DISCUSSION TITLE
Can I set the model directory to another folder?

### DISCUSSION THREAD

**Discussion by Cubey42** [anapnoe/stable-diffusion-webui-ux#66] [General]
I don't want to have another model for a1111 and webui-ux, is there anyway to point it to a different model dir?

**Comment by Skuwwy** [comment]
You can use symlinks instead of copy pasting the files, or use args in the launcher such as _--ckpt-dir D:/"SD"/Models/SD/_ and point to your existing directory with all the models and extensions
